### DISCUSSION TITLE
global.sls BPF Syntax Multiple IP's

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#10782] [2.4] [answered]
Is this the correct syntax for filtering out individual IP's from suricata and zeek for all minions?

/opt/so/saltstack/local/pillar/global.sls

nids:
  bpf:
     - not host 10.12.7.221
     - not host 10.13.7.222
     - not host 10.14.7.222
zeek:
  bpf:
     - not host 10.12.7.221
     - not host 10.13.7.222
     - not host 10.14.7.222

**Comment by Ryan8bit** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/bpf.html
https://www.youtube.com/watch?v=uamNOjtUi4Y&feature=youtu.be
Use "and" to separate each condition.
